tools/lab_dispatch/build_install_deb.py

```diff
@@ -16,37 +34,62 @@ def parse_args():
 
 
 def prepare_repo(repo_root, package_path):
-    shutil.move(package_path / "debian", repo_root)
+    shutil.move(str(package_path / "debian"), str(repo_root))
```

Comment:
Is Python 3.8 used on our runners?

Reply:
This runs on the DUT, which are bionic+, so yes, 3.8 it is (but our agents do run python3.8 as well)

Reply:
Thanks. It seems `shutil.move` started accepting `pathlib.Path`s as arguments starting from 3.9, so we can't avoid the conversion to strings.